src/components/AddNewElemModal/Modal.jsx

```diff
@@ -37,7 +37,7 @@ const Modal = ({ onNewItemHandler, onCloseModal }) => {
 
   return (
     <ModalPortal onClick={closeModal}>
-      <form onSubmit={addNewToDoItemHandler}>
+      <form onSubmit={addNewToDoItemHandler} className={classes.mobile_modal}>

```

Comment:
A che serve questa classe? Non la trovo in nessun file

Reply:
tolta!